src/ts/core/roam/references.ts

```diff
@@ -44,3 +44,22 @@ export const openParentPage = (shiftKey: boolean = false) => {
 
     Mouse.leftClick(parentLink, shiftKey)
 }
+
+const getMentionsButton = (blockElement: BlockElement | null): HTMLElement | null => {
+    const blockMentionsButton = blockElement
+        ?.closest(Selectors.blockBulletView)
+        ?.querySelector('.block-ref-count-button')
+    if (blockMentionsButton) {
+        return blockMentionsButton as HTMLElement
+    }
+
+    const sidePanel = blockElement?.closest(Selectors.sidebarPage)
+    return sidePanel?.querySelector('button.bp3-button') as HTMLElement
+}
+
+export const openMentions = (shiftKey: boolean = false) => {
+    const mentionsButton = getMentionsButton(RoamBlock.selected().element)
+    if (mentionsButton) {
+        Mouse.leftClick(mentionsButton, shiftKey)
+    }
+}
```

Comment:
Shift+cmd click allows opening the mentions directly. so this can be simplified

Reply:
Like shift+cmd clicking on the block? Shift+cmd click isn't doing anything for me right now

Reply:
Oh I misinterpreted what this does. By analogy to "Shift 1" I thought this would open page mentions when you are navigating query/mentions block.  
But this opens the block references or page references if you're in sidebar, right?  If so maybe we should consider a different default shortcut?

Reply:
Oh right, I forgot to include the demo:

If the block has mentions, this opens the block's mentions

![open mentions](https://user-images.githubusercontent.com/1150079/95773102-b22c8800-0c72-11eb-8aca-b8272ca8c51c.gif)

If the block has no mentions, this open's the page's mentions

![Mentions Page](https://user-images.githubusercontent.com/1150079/95773547-86f66880-0c73-11eb-9f8b-51a18c64bd28.gif)

So this is kind of like "open the mentions for whatever is selected"

Reply:
Nice! would be also great if it were to work for the pages in query/mentions :). but we can do that later